starters/express-typescript-typeorm-postgres/src/main.ts

```diff
@@ -2,11 +2,26 @@
 require('dotenv').config();
 import { bootstrapApp } from './bootstrap-app';
 import { dataSource } from './datasource';
+import cors from 'cors';
 
 const PORT = process.env.PORT || 3333;
 
 const app = bootstrapApp();
 
+const whitelist: string[] = JSON.parse(process.env.CORS_ALLOWED_ORIGINS || []);
+
+const corsOptions = {
+  origin: (origin, callback) => {
+    if (whitelist.indexOf(origin) !== -1) {
+      callback(null, true);
+    } else {
+      callback(new Error());
+    }
+  },
+};
+
+app.use(cors(corsOptions));
```

Comment:
Is there a default setting with cors? With "same-origin-only"? I think we should use that configuration if there are no allowed origins provided in the environment.